### PR TITLE
VGB-15: Implement POST /login

### PR DESCRIPTION
**Pull request by Jonathan-Bonnin**

### CODE REVIEW

File: GiTinder/Controllers/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            if (user == null)
+            {
+                return NotFound();
+            }
+
+            _context.Users.Remove(user);
+            await _context.SaveChangesAsync();
+
+            return Ok(user);
+        }
```
Comment (adamgyulavari) [inline GiTinder/Controllers/UsersController.cs]: @Jonathan-Bonnin remove unnecessery endpoints from controller

File: GiTinder/Controllers/UsersController.cs [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+using Microsoft.EntityFrameworkCore;
+using GiTinder.Data;
+using GiTinder.Models;
+using System.Net.Http;
+using System.Net;
+using GiTinder.Services;
+
+namespace GiTinder.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (adamgyulavari) [inline GiTinder/Controllers/UsersController.cs]: @Jonathan-Bonnin this is not needed here

File: GiTinder/Controllers/UsersController.cs [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        private readonly UserServices _userServices;
+
+        public UsersController(GiTinderContext context, UserServices userServices)
+        {
+            _context = context;
+            _userServices = userServices;
+        }
+        
+        // POST: /api/login
+        [HttpPost("/api/login")]
```
Comment (adamgyulavari) [inline GiTinder/Controllers/UsersController.cs]: @Jonathan-Bonnin this should be `/login`

File: GiTinder/Services/UserServices.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            string[] tokenChars = { "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p",
+                "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9"};
+            Random random = new Random();
+
+            do
+            {
+                for (int i = 0; i < 28; i++)
+                {
+                    token += tokenChars[random.Next(0, tokenChars.Length)];
+                }
```
Comment (adamgyulavari) [inline GiTinder/Services/UserServices.cs]: @Jonathan-Bonnin you can use guid for this

File: GiTinderTests/Models/UserTests.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
```
Comment (adamgyulavari) [inline GiTinderTests/Models/UserTests.cs]: @Jonathan-Bonnin remove GiTinderTest project

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by Jonathan-Bonnin** [comment]
@adamgyulavari 

- removed unneeded endpoints
- token is created using Guid
- duplicated Test folder removed
- endpoint changed from "/api/login" to "/login"
- merge conflicts resolved

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**